earth2mip/initial_conditions/gfs.py

```diff
@@ -419,10 +426,11 @@ def _get_gfs_name_dict() -> Dict[str, str]:
 
 
 class DataSource(base.DataSource):
-    def __init__(self, channels: List[str]) -> None:
+    def __init__(self, channels: List[str], download_ic=True) -> None:
         lookup = _get_gfs_name_dict()
```

Comment:
Unrelated to this MR, but the way the dict is created is just so prone to error it hurts: `dict(zip(channels, gfs_channels))`

Reply:
Agreed, but beyond the scope of this PR to fix.